packages/design-system-react/src/components/AvatarFavicon/AvatarFavicon.test.tsx

```diff
@@ -65,7 +66,7 @@ describe('AvatarFavicon', () => {
         <AvatarFavicon
           name="ACME"
           src={src}
-          imageProps={{ 'data-testid': 'img', id: 'img-id' } as any}
+          imageProps={{ 'data-testid': 'img', id: 'img-id' }}
```

Comment:
If we have to use type aliases here so will engineers at the platform end unfortunately data-testids are not included in the react types so we will have to directly specify it for all of our components